app/src/main/java/com/picone/go4lunch/presentation/ui/restaurant/RestaurantDetailFragment.java

```diff
@@ -28,16 +37,25 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
         mBinding = FragmentRestaurantDetailBinding.inflate(inflater, container, false);
-        initRecyclerView();
         showAppBars(false);
+        initRecyclerView();
+        populateView();
         return mBinding.getRoot();
     }
 
+    private void populateView() {
+        mBinding.restaurantNameDetailTextView.setText(restaurant.getName());
+        mBinding.foodStyleAndAddressDetailTextView.setText(restaurant.getFoodType()
+                .concat(" - ").concat(restaurant.getAddress()));
+    }
+
     private void initRecyclerView() {
-        RecyclerView.LayoutManager linearLayoutManager = new LinearLayoutManager(getContext());
-        mBinding.recyclerViewRestaurantDetail.setLayoutManager(linearLayoutManager);
+        mAdapter = new ColleagueRecyclerViewAdapter(mInterestedUsers);
+        mBinding.recyclerViewRestaurantDetail.setLayoutManager(new LinearLayoutManager(getContext()));
         mBinding.recyclerViewRestaurantDetail.setAdapter(mAdapter);
     }
 
-
+    static void updateRestaurantDetailUi(Restaurant restaurant) {
```

Comment:
Static for what ?

Reply:
To pass restaurant from RestaurantListfragment to RestaurantDetailFragment. Do i have to pass the object with an event or something like this?

Reply:
Use a livedata